docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging_JDBC_Store_for_Messaging_Journal.adoc

```diff
@@ -23,24 +23,7 @@ server can use it for its JDBC store with the operation:
 Artemis JDBC store uses SQL commands to create the tables used to
 persist its information. +
 These SQL commands may differ depending on the type of database. The SQL
-commands used by the JDBC store are located in the file at:
-
-....
-$JBOSS_HOME/modules/system/layers/base/org/wildfly/extension/messaging-activemq/main/database/journal-sql.properties
-....
-
-By default, "vanilla" SQL commands are used to communicate with the
-database. However some databases requires specific commands to create
-table, update content, etc.
-
-The journal-sql.properties can also specify these provider-specific
-commands. You can customize them by adding a suffix to the vanilla SQL
-properties.
-
-The suffix is determined based on information from the JDBC driver and
-the connection metadata. If the type of database is not supported by the
-code, you can specify it explicitly with the journal-database property
-on the server resource.
+commands used by the JDBC store are located in artemis-jdbc-store jar.
```

Comment:
It seems that mentioned commands are stored in `modules/system/layers/base/org/apache/activemq/artemis/main/artemis-jdbc-store-${ARTEMIS_VERSION}.jar/journal-sql.properties`. But this PR removes details about journal-sql.properties file. Is this change related to https://issues.redhat.com/browse/WFLY-14087 jira and related pool configuration?

Reply:
Given that this file is provided by Artemis now and is not meant to be overriden by WildFly, I'd rather remove that incorrect documentation

Reply:
Sure, current documentation is incorrect. But new documentation suggest that users should check artemis-jdbc-store jar for SQL commands, but doesn't provide details where these commands could be found in .jar file.